ui/src/app/utils/access-utils.tsx

```diff
@@ -134,6 +135,14 @@ export const getAccessModuleConfig = (moduleName: AccessModule): AccessModuleCon
       refreshAction: async () => await profileApi().syncComplianceTrainingStatus(),
     })],
 
+    [AccessModule.CTCOMPLIANCETRAINING, () => ({
+      moduleName,
+      isEnabledInEnvironment: enableComplianceTraining,
+      DARTitleComponent: () => <div>Bypass {environment.displayTag} <AoU/> research Controlled Tier training </div>,
```

Comment:
This looks a little odd to me.  
<img width="428" alt="Screen Shot 2021-11-01 at 12 26 49 PM" src="https://user-images.githubusercontent.com/2701406/139705858-c75a1712-8b13-4ea0-b28a-1e516599bf13.png">

How about `Bypass <AoU/> research Controlled Tier training [Local->Test environment]` ?

Reply:
How about [Local->Test enivornment] ByPass <AoU/> research Controlled Tier training (just like self bypass?)

Reply:
@NehaBroad sgtm

Reply:
could you add the brackets as well?

Reply:
Its now 
<img width="627" alt="Screen Shot 2021-11-01 at 3 32 07 PM" src="https://user-images.githubusercontent.com/34481816/139730265-319c09fe-68f8-4725-bc0b-d31f20d4a5ec.png">